### PR TITLE
Connect clean up

### PR DESCRIPTION
**Pull request by filipenevola**
Cleans up `connect` from code & readme.

Meteor 3 replaces `connect` with `express` but there are still some references in the code.

- [x] Remove unused connect dependency
- [x] Replaces explanation about how to use the module directly
- [x] Review webapp_server.js mentions to connect
- [x] Review `import connectRoute from 'connect-route'` in the docs example
- [x] Keep old names available but using express
- [x] Change expressXyz to only Xyz
- [x] Review and answer this [thread](https://forums.meteor.com/t/what-impact-does-webapp-transition-from-connect-to-express-have-on-community-packages-that-use-it/60541)

### CODE REVIEW

**Review by Grubba27 — CHANGES_REQUESTED** [review]
From this comment [here](https://forums.meteor.com/t/what-impact-does-webapp-transition-from-connect-to-express-have-on-community-packages-that-use-it/60541/5?u=grubba) on Forums I would ask as well to bring back the `connect*` handlers but that do throw a warning or a log.

**Comment by Grubba27** [comment]
> One idea would be to use just `Webapp.handlers` going forward so if for some reason express gets swapped with something else in the future you don’t need to rename anything 🙂

That sounds even better. What do you think @filipenevola ?

We depreciate but not remove `connectHandlers` and add and maintain just `handlers`

**Comment by filipenevola** [comment]
Sure, I updated the checklist.

**Comment by filipenevola** [comment]
@jamauro @Grubba27 code changes are done.

Now just missing to review all the mentions of connect in the code.

**Comment by filipenevola** [comment]
@Grubba27 just so you know, 2 tests of webapp package are failing on my machine, the same happens in release-3.0 branch.

Also, FYI I'm running on ARM (Apple M2).

**Review by denihs — APPROVED** [review]

File: packages/webapp/webapp_server.js [hunk 1]
```diff
@@ -1346,16 +1345,21 @@ async function runWebAppServer() {
...
+    rawConnectHandlers: rawExpressHandlers,
+    rawHandlers: rawExpressHandlers,
     httpServer: httpServer,
     expressApp: app,
     // For testing.
-    suppressConnectErrors: function() {
-      suppressConnectErrors = true;
-    },
+    suppressConnectErrors: suppressErrors,
+    suppressExpressErrors: suppressErrors,
```
Comment (radekmie) [inline packages/webapp/webapp_server.js:1362]: Would it make sense to add runtime deprecation warnings?

**Review by radekmie — APPROVED** [review]

File: packages/webapp/webapp_server.js (hunk 1 above)
Comment (filipenevola) [reply packages/webapp/webapp_server.js:1362]: IMO yes. Do you agree?

Comment (radekmie) [reply packages/webapp/webapp_server.js:1362]: Yeah, let's do that. While they have very similar APIs, there can be some nuanced differences.

**Review by Grubba27 — APPROVED** [review]

File: packages/webapp/webapp_server.js (hunk 1 above)
Comment (radekmie) [reply packages/webapp/webapp_server.js:1362]: https://github.com/meteor/meteor/blob/e0631738f2a8a914d8a50b1060e8f40cb0873680/packages/meteor/helpers.js#L211-L222

Comment (filipenevola) [reply packages/webapp/webapp_server.js:1362]: done!

**Comment by filipenevola** [comment]
@Grubba27 if all tests pass (I just synced with release-3.0) this PR is ready to be merged.

**Pull request merged**